components/Button.tsx

```diff
@@ -0,0 +1,14 @@
+import styles from "@/styles/Button.module.css";
+import { textFont } from "@/pages/_app";
+
+const Button = ({ text }: { text: string }) => {
```

Comment:
It is good for now. But consider what if the props are not only about the text but also with icons or even no text props.

Reply:
![image](https://user-images.githubusercontent.com/73293255/218648404-4cdeb339-7f8a-4f26-834f-30bd75ecea90.png)
Can we do it this way？